pkgs/by-name/dp/dprint/package.nix

```diff
@@ -20,11 +21,19 @@ rustPlatform.buildRustPackage (finalAttrs: {
     owner = "dprint";
     repo = "dprint";
     tag = finalAttrs.version;
-    hash = "sha256-6AgbKH5f7N/yYqq7KBVHOqYbyuZkjFSaYwZwIXsgd9o=";
+    hash = "sha256-Lt6CzSzppu5ULhzYN5FTCWtWK3AA4/8jRzXgQkU4Tco=";
   };
 
+  cargoPatches = [
```

Comment:
My apologies for submitting the patch without a draft PR. It appears the issue was resolved by setting an environment variable, making the patch and additional inputs unnecessary.

https://github.com/NixOS/nixpkgs/pull/422610

Reply:
No problem. I will use this pr to experiment with something else and close it, you can unsubscribe from this PR